src/components/accordion-item/accordion-item.scss

```diff
@@ -102,6 +102,7 @@
     items-center
     duration-150
     ease-in-out;
+  word-break: break-word;
```

Comment:
We have this mixin that we could use here: https://github.com/Esri/calcite-components/blob/14fff178e50a62cb58fc66aa43b7b2332d4d3eff/src/assets/styles/includes.scss#L22-L25